sdk/samples/iot/aziot_esp32/New-ArduinoZipLibrary.ps1

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
```

Comment:
Can we have a single script that is linked to in each of these sample folders?  At first glance they look the same. Is there a scenario we want them to differ?

Reply:
Yes, I have plans to consolidate these scripts and other common code as next step.